### PR TITLE
This series makes the build of the whole layer more reliable

### PR DESCRIPTION
**Pull request by fmoessbauer**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ variables:
   SSTATE_LOCATION: "${RUNNER_SSTATE}/${CI_PROJECT_NAME}"
   SSTATE_MIRRORS: "file://.* ${SSTATE_LOCATION}/PATH;downloadfilename=PATH"
   SSTATE_MAX_AGE: "3d"
+  DISTRO_APT_PREMIRRORS: 'http://deb\.debian\.org https://debian.inf.tu-dresden.de\n'
```
Comment (jan-kiszka) [inline .gitlab-ci.yml]: Why? I know reasons, but they should go into the commit message.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ variables:
 
 .kas-build:
   stage: build
-  image: ghcr.io/siemens/kas/kas-isar:3.0.2
+  image: ghcr.io/siemens/kas/kas-isar:3.1
```
Comment (jan-kiszka) [inline .gitlab-ci.yml:25]: What about the kas-container copy in this layer?

Comment (fmoessbauer) [reply .gitlab-ci.yml:25]: Good catch. Fixed.

File: .gitlab-ci.yml (hunk 1 above)
Comment (fmoessbauer) [reply .gitlab-ci.yml]: Added the reason.

File: .gitlab-ci.yml (hunk 2 above)
Comment (jan-kiszka) [reply .gitlab-ci.yml:25]: Then maybe also rename the subject as you are no longer only targeting "ci:".

File: .gitlab-ci.yml (hunk 1 above)
Comment (jan-kiszka) [inline .gitlab-ci.yml]: If that only affects our CI setup, then better make this a CI variable that is using this value only in our local project settings.

**Comment by jan-kiszka** [comment]
Looks good otherwise and works as well (on IOT2050).

**Review by jan-kiszka — APPROVED** [review]

**Pull request merged**